.github/workflows/dispatch-release-8-2.yaml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   run-release:
     name: "Release ${{ github.event.client_payload.releaseVersion }}"
-    uses: ./.github/workflows/release.yml
+    uses: camunda/zeebe/.github/workflows/release.yml@stable/8.2
```

Comment:
🔧 I think we also need to add a new workflow `dispatch-release-8-3.yaml` for the new 8.3 minor going forward.

Maybe we need to rephrase this [form](https://github.com/zeebe-io/zeebe-engineering-processes/blame/master/src/main/resources/release/forms/post_release_update_dry_stable_release_workflow.form#L20) a bit, highlighting the current released one needs and update  a new one for the new development version needs to created 🤔

Reply:
Ah, I wasn't aware of it. I was wondering why the workflow already existed to be honest :sweat_smile:

Reply:
I'll update the release process as well :+1: